pybossa/view/projects.py

```diff
@@ -3215,7 +3216,8 @@ def ext_config(short_name):
     project, owner, ps = project_by_shortname(short_name)
     sanitize_project, _ = sanitize_project_owner(project, owner, current_user, ps)
 
-    ext_conf = project.info.get('ext_config', {})
+    project.info[u'ext_config'] = project.info.get(u'ext_config', {}) 
+    ext_conf = project.info.get(u'ext_config', {})
```

Comment:
above line 3219 can be removed. also remove unicode tag can be removed from key name. basically, reverting to original code.